ams/api/src/main/java/com/netease/arctic/ams/api/properties/CatalogMetaProperties.java

```diff
@@ -48,11 +48,7 @@ public class CatalogMetaProperties {
   // Deprecated from version v0.4.0, use KEY_WAREHOUSE
   @Deprecated public static final String KEY_WAREHOUSE_DIR = "warehouse.dir";
   public static final String KEY_WAREHOUSE = "warehouse";
-  /**
-   * @deprecated since 0.6.0, will be removed in 0.7.0; use {@link
-   *     CatalogMetaProperties#KEY_TABLE_FILTER} instead.
-   */
-  @Deprecated
+
```

Comment:
Should we change the property name to `database-filter`, to keep the same pattern with `table-filter`?

Reply:
BTW, we may add the new property into user docs.

Reply:
Use `database.filter-regular-expression` just considering compatibility. 
Keeping the same style is more suitable for subsequent use, and `database.filter-regular-expression` has been deprecated in 0.6, so I agree to change it to `database-filter`.

Done.